minimal/docker-compose.yml

```diff
@@ -53,4 +53,13 @@ services:
     volumes_from:
       - {{ project_name }}_web
     ports:
-      - "0.0.0.0:80:80"
\ No newline at end of file
+      - "0.0.0.0:80:80"
+
+  {{ project_name }}_certbot:
+    restart: always
+    build: ./compose/certbot/production
+    depends_on:
+      - {{ project_name }}_web
+    volumes_from:
```

Comment:
I need access to `{{ project_name}}_web`'s files since that's what nginx is serving. That way it simply add .staticfiles to the certbot container which won't serve our purpose. We could also add a second upstream server just for certbot. That way we wouldn't even need access to `{{ project_name }}_web`.